src/componets/SearchNav/dropdown.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  DropDownContainer,
+  DropDownHeader,
+  DropDownListContainer,
+  DropDownList,
+  ListItem,
+} from './styles';
+
+import Icon from '../Icon/index';
+
+function Dropdown({
+  placeholder,
+  options,
+  color,
+  background,
+  haveIcon,
+  setData,
+  isActive,
+}) {
+  const dropDownRef = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const onOptionClicked = (index) => () => {
+    setData(index);
+  };
+  const toggling = () => setIsOpen(!isOpen);
+  useEffect(() => {
+    const pageClickEvent = (e) => {
+      const {
+        bottom,
+        height,
+        left,
+        right,
+      } = e.target.getBoundingClientRect();
+      if (bottom && height && left && right) setIsOpen(false);
+    };
+    if (isOpen) window.addEventListener('click', pageClickEvent);
+    return () => {
+      window.removeEventListener('click', pageClickEvent);
+    };
+  }, [isOpen]);
+  return (
+    <DropDownContainer>
+      <DropDownHeader
+        onClick={toggling}
+        color={color}
+        background={background}
+        selected={isActive}
+      >
+        {placeholder}
+        {isOpen
+          ? <Icon name="UpIcon" fill={color} width="15px" height="15px" margin="0 0 0 5px" selected={isActive} />
+          : <Icon name="DownIcon" fill={color} width="15px" height="15px" margin="0 0  0 5px" selected={isActive} />}
+      </DropDownHeader>
+      {isOpen && (
+        <DropDownListContainer ref={dropDownRef}>
+          <DropDownList>
+            {
+              options.map((option, index) => (
+                <ListItem
+                  onClick={onOptionClicked(index)}
+                  key={Math.random()}
+                  selected={option.active}
+                >
+                  {haveIcon ? (
+                    <Icon
+                      name={option.name}
+                      width="15px"
+                      height="15px"
+                      margin="0 10px 0 0"
+                      selected={option.active}
+                      fill={color}
+                    />
+                  ) : ''}
```

Comment:
```suggestion
                  {haveIcon && (
                    <Icon
                      name={option.name}
                      width="15px"
                      height="15px"
                      margin="0 10px 0 0"
                      selected={option.active}
                      fill={color}
                    />
                  )}
```
you can use an and operator in this case you want just render one case